File: src/main/java/net/flyingfishflash/loremlist/api/data/response/LrmListItemAddedResponse.java
package net.flyingfishflash.loremlist.api.data.response;

import java.util.List;
import net.flyingfishflash.loremlist.domain.SuccinctLrmComponent;

public record LrmListItemAddedResponse(
    String componentName, List<SuccinctLrmComponent> associatedComponents) {}

File: src/main/kotlin/net/flyingfishflash/loremlist/api/data/response/LrmListItemAddedResponse.kt
package net.flyingfishflash.loremlist.api.data.response

import kotlinx.serialization.Serializable
import net.flyingfishflash.loremlist.domain.SuccinctLrmComponent

@Serializable
data class LrmListItemAddedResponse(val componentName: String, val associatedComponents: List<SuccinctLrmComponent>)
